services/docker-compose.yml

```diff
@@ -390,7 +390,7 @@ services:
   invitations:
     image: ${DOCKER_REGISTRY:-itisfoundation}/invitations:${DOCKER_IMAGE_TAG:-latest}
     init: true
-    hostname: "{{.Node.Hostname}}-{{.Task.Slot}}"
+    hostname: "inv-{{.Node.Hostname}}-{{.Task.Slot}}"
```

Comment:
ohlala... is this for the RabbitMQ queue names? maybe we should add the application name in the client side to alleviate this stuff.

Reply:
i recall discussing with @YuryHrytsuk about the limits of HOSTNAME. That is why i just add a small prefix